app/src/main/java/com/github/fribourgsdp/radio/data/Playlist.kt

```diff
@@ -120,6 +153,20 @@ data class Playlist (override var name: String, var genre: Genre) : Nameable {
         return db.registerPlaylist(this)
     }
 
+    /**
+     * @return true if all songs have lyrics
+     */
+    fun allSongsHaveLyrics(): Boolean{
+        return songs.all { it.lyrics != "" }
+    }
+
+    /**
+     * @return true if no song has lyrics
+     */
+    fun noSongHaveLyrics(): Boolean{
+        return songs.all { it.lyrics == "" || it.lyrics == "---"}
+    }
+
```

Comment:
Why do these 2 methods not consider the same criteria of _having no lyrics_ ? it means that if every song in a playlist have lyrics = --- , it will consider that all songs in pl have lyrics ?

Reply:
Yes indeed. I created these method in order to be able to test whether we should make a musixmatch call or not. In this context, having lyrics or having no lyrics from musixmatch results in the same behavior. For me, the second definition makes more sense and another name should be found for the first function. In any cases, I was planning on using the method of song that returns whether it has lyrics or not that you told me you would implement.

Reply:
I created a new one so that the methods are clearer.

Reply:
Yeah that's better